docs/framework/react/quick-start.md

```diff
@@ -3,4 +3,53 @@ title: Quick Start
 id: quick-start
 ---
 
-# TODO
\ No newline at end of file
+The basic react app example to get started with the Tanstack react-store.
+
+```tsx
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { useStore } from "@tanstack/react-store";
+import { Store } from "@tanstack/store";
+
+export const store = new Store({
+  value1: 0,
+  value2: 0,
```

Comment:
Instead of `value1` and `value2` can we call this maybe `cats` and `dogs` and reframe this example as a poll-taking demo?